### PR TITLE
feature: Extend build process, integrate main page / ida page switch, streamline content files

### PR DESCRIPTION
**Pull request by coldice**
This PR extends the build scripts to allow building the main page and the ida page from the same branch.
Further readme and instruction updates will be added.

* [ ] Small language/grammar fixes
* [ ] Small content fixes 
* [ ] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [x] Other

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
-	npm install
+	npm ci
 	npm run build
 
+build-ida-website: 
+	echo "Use ida customisations"
+	cp -rf ida-customisations/ ./
+	echo "Build website"
+	npm ci
+	npm run build
```
Comment (xavierlepretre) [inline Makefile:12]:
```suggestion
	build-website
```

File: package.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "serve": "vuepress dev --no-cache",
     "build": "vuepress build --no-cache",
+    "build:ida": "make build-ida-website",
```
Comment (xavierlepretre) [inline package.json:13]: Can there be something similar like `serve:ida`?

**Comment by coldice** [comment]
@xavierlepretre I have added a few QoL functions and prepared a doc for the use. Can you please:

* Review [TECHNICAL-SETUP.md](https://github.com/cosmos/sdk-tutorials/pull/1179/files#diff-150eb7e65c21a908a32b6c1fa85d694f980b05b55d90104f715ad60204727e09).
* Try to break this, **following the instructions**.
* Let me know if you see anything amiss.

TL;DR: Added filesystem switch scripts. Please try `npm run switch-ida` and `npm run switch-main`.

**Comment by xavierlepretre** [comment]
> * Review [TECHNICAL-SETUP.md](https://github.com/cosmos/sdk-tutorials/pull/1179/files#diff-150eb7e65c21a908a32b6c1fa85d694f980b05b55d90104f715ad60204727e09) (just tech).
> * Try to break this, **following the instructions**.
> * Let me know if you see anything amiss.

The description looks good to me.

So I:

* `serve` as checked out -> I got the main website.
* I ran `switch-ida` and I only get 2 files that were copied, `.vuepress` thingy is missing. Could it be because it is a hidden file?
* I updated What's next.
* I ran `switch-main`, my change is correctly reflected in the IDA's what's next, but the regular what's next and Readme are still the IDA versions, they were not restored.
